lsp/src/main/scala/org/ensime/lsp/rpc/companions.scala

```diff
@@ -0,0 +1,136 @@
+// Copyright: 2010 - 2017 https://github.com/ensime/ensime-server/graphs
+// License: http://www.gnu.org/licenses/gpl-3.0.en.html
+package org.ensime.lsp.rpc.companions
```

Comment:
why are these called companions? That confuses me, because "companion" has a scala meaning.

Reply:
Originally there were `*Companion` classes. So I named file with them as `companions` :)